### ISSUE TITLE
fanfcion.net ignoring all paragraphs that are aligned to center

### ISSUE THREAD

**Issue by dipu-bd** [dipu-bd/lightnovel-crawler#564]
**Describe the bug**
I found a issue on a fanfcion.net story. I do not know if it occurs in other sources as well.
The fetcher is ignoring all paragraphs that are aligned to center.

As an exemple:
On this story https://www.fanfiction.net/s/13277163/1/A-Gamers-Guide-to-Necromancy
In the very start there a various paragraphs like these:
```
<p style='text-align:center;'><strong>Welcome! You have been chosen as the Host of the Gamer System!</strong></p>
```
However they are not present on the .json after its downloaded.

**Let us know**
- Novel URL: https://www.fanfiction.net/s/13277163/1/A-Gamers-Guide-to-Necromancy

**Comment by fof300f** [comment]
```
for p in contents.select('p[style="text-align:center;"]'):
            p.decompose()

```

**Comment by dipu-bd** [comment]
the reason to remove all center aligned p tags was to remove unwanted ad texts. but now that it is removing chapter content too, we have to find a better way to remove ad texts

**Comment by fof300f** [comment]
There should be a reason why it has been added in the first place, I'll have a look at it

**Comment by dipu-bd** [comment]
@fof300f do you want to work on this issue?

**Comment by fof300f** [comment]
sure

**Comment by fof300f** [comment]
Well, I've been looking around a bit and I can't find these aforementioned ad's. I've downloaded the novel and searched for any ad's but they don't seem to occur.

**Comment by fof300f** [comment]
I couldn't find these ad's even after crawling one of the longest novels on the site. Seems they changed their advertising concept... been to the site without adblocker, didn't see anything either. Would remove the mentioned code and monitor the stuff for a while

**Comment by dipu-bd** [comment]
ok, I will be waiting for your PR

**Comment by dipu-bd** [comment]
solved by #565
